src/frontend/shortlist/package.json

```diff
@@ -11,6 +11,8 @@
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore"
   },
   "dependencies": {
+    "@popperjs/core": "^2.11.6",
+    "bootstrap": "^5.2.2",
```

Comment:
Just noting for all that we've added "bootstrap" dependency with this particular PR... monitoring this new dependency over the next few sprints to make sure it doesn't interfere with expected vue behaviors.  if something odd happens while you're trying to do things like v-bind or v-model, this might be the culprit

Reply:
Added as an Important comment in the PR description for members.